SafeguardSessions/README.md

```diff
@@ -16,6 +16,8 @@
 
 The aim of One Identity Safeguard Power BI Connector (Power BI Connector) is to provide a solution for customers to visualize their audit data captured by [One Identity Safeguard for Privileged Sessions] (SPS) in a highly configurable way compared to the on-box reporting system of SPS. The Power BI Connector uses basic authentication with the local login method to connect to SPS, and imports sessions metadata that matches the criteria specified in the Power BI Connector's input parameters. After successful data retrieval, the Power BI Connector returns the following two tables.
 
+The Power BI Connector is usable in Power BI Desktop and Power BI Service.
```

Comment:
One question I have is: is it self-explanatory how to use the connector in the service? If I know you need some kind of gateway in the Power Bi Service, I don't think it is necessary to store the SPS certificate locally (I'm not sure about this).

I think we should update the Installation and usage part.

Reply:
The SPS certificate is a must have in all cases.

Using the service is not that straightforward. I'll think about it. Maybe we can reference the official documentation.